internal/pkg/cli/app_show.go

```diff
@@ -138,7 +138,7 @@ func (o *showAppOpts) retrieveData() (*describe.WebApp, error) {
 		if err == nil {
 			routes = append(routes, &describe.WebAppRoute{
 				Environment: env.Name,
-				URL:         fmt.Sprintf("%s/%s", webAppURI.DNSName, webAppURI.Path),
+				URL:         fmt.Sprintf("%s", webAppURI.String()),
```

Comment:
Hm, this might not work the way we expect since this will be a colorized. string. 

I have a suggestion :)

Reply:
Can this `fmt.Sprintf` be removed?

Reply:
yup. done!